lib/peeps.rb

```diff
@@ -0,0 +1,32 @@
+class Peeps
+
+  require 'pg'
+
+  attr_accessor :message , :date 
+
+  def initialize(message:, date:)
+    @date = date
+    @message = message 
+  end 
+
+  def self.list 
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+  end
+    result = connection.exec("SELECT * FROM peeps;")
+    result.map { |peeps| Peeps.new(message: peeps['message'],date: peeps['date']) }
+  end
+ 
+  def self.create(message:, date:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
```

Comment:
This piece of code appears twice in this file. How might you remove this duplication?